draft-ietf-privacypass-auth-scheme.md

```diff
@@ -205,12 +205,19 @@ practice. Challenges with redemption_context values of invalid lengths MUST be i
 
 - "origin_info" is an ASCII string that is either empty, or contains one or more
 origin names that allow a token to be scoped to a specific set of origins. Each
-origin name uses the format of the authority portion of a URI as defined in
-{{Section 3.2 of URI}}. The string is prefixed with a 2-octet integer indicating
-the length, in network byte order. If empty, any non-origin-specific token can be
-redeemed. If the string contains multiple origin names, they are delimited with
-commas "," without any whitespace. If this field is not empty, the Origin MUST
-include its own name as one of the names in the list.
+origin name uses the format of a server name described below. The string is
+prefixed with a 2-octet integer indicating the length, in network byte order.
+If empty, any non-origin-specific token can be redeemed. If the string contains
+multiple origin names, they are delimited with commas "," without any whitespace.
+If this field is not empty, the Origin MUST include its own name as one of the
+names in the list.
+
+The server names used in the "issuer_name" and "origin_info" fields are ASCII strings
```

Comment:
I think this would read better if we put this description in the terminology section and then referenced it in these two spots.

Reply:
@chris-wood I tried that, but it felt awkward in terminology.

I've now updated the sections for challenges and redemption to include more subsections, and gave this its own subsection to refer to. I think this should make it cleaner to read. Thoughts?

Reply:
See https://ietf-wg-privacypass.github.io/base-drafts/tfpauly-patch-34/draft-ietf-privacypass-auth-scheme.html

Reply:
Love it!